File: lab-3/tema-lab-2/t2-java/src/main/java/Timer.java
public class Timer {
    private static long start = 0;

    public static void Start() {
        start = System.currentTimeMillis();
    }

    public static long Stop() {
        return System.currentTimeMillis() - start;
    }

    public static void PrintState() {
        System.out.println(System.currentTimeMillis() - start);
    }

    public static void PrintPeriod() {
        long stop = System.currentTimeMillis();
        System.out.println(stop - start);
        start = stop;
    }
}

File: lab-2/tema-lab-1/t1-cpp-2/src/timer.cpp
#include "timer.h"

void Timer::start() {
    timerStart = clock();
}

long Timer::stop() const {
    float seconds = ((float(clock() - timerStart)) / CLOCKS_PER_SEC);
    long milliseconds = long(seconds * 1000);
    return milliseconds;
}

void Timer::printState() const {
    cout << long(((float(clock() - timerStart)) / CLOCKS_PER_SEC) * 1000);
}

void Timer::printPeriod() {
    clock_t stop = clock();
    cout << long(((float(stop - timerStart)) / CLOCKS_PER_SEC) * 1000);
    timerStart = stop;
}
